source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Benefit to oVirt users
+oVirt users may now analyze and see thier network topology via the Skydive UI. 
+Skydive allows the users to see the result of their configuration done via oVirt.
+They may capture traffic from VM1 to VM2 for example or monitor the traffic between two VMs or hosts. 
```

Comment:
suggest "For example, administrators can capture traffic from VM1 to VM2, or monitor the traffic between VMs or hosts."

Reply:
done